ansible/roles/rox/tasks/main.yml

```diff
@@ -13,8 +13,9 @@
   shell: composer config --global github-protocols https
 
 - name: Install Rox dependencies | Composer
-  composer: command=install working_dir={{ rox.root_folder }}
-
+  # ansible 1.7.2 (latest version with apt-get) doesn't accept this syntax below
+  # composer: command=install working_dir={{ rox.root_folder }}
+  command: composer install --working-dir {{ rox.root_folder }}
```

Comment:
ansible's version available in apt-get doesn't support "composer", so the same stuff needs to be done thru "command"